css/style.css

```diff
@@ -15,74 +16,82 @@ html, body{
   src: url('../fonts/Saurabh-Regular.ttf');
 } */
 
-.page{
+.page {
   padding: 10px;
-  position:relative;
-  background: linear-gradient(to right,#fff, #fff);
-  padding:0px;
+  position: relative;
+  background: linear-gradient(to right, #fff, #fff);
+  padding: 0px;
```

Comment:
Would you mind removing the changes that happened due to your code-formatter? It will later be difficult to debug what was changed in this PR because of this additional changes

Reply:
So I should delete this re-fork , clone it, edit it without prettier and re-push and then PR. It was my first PR and I am still a beginner learning the git so please be kind enough and bare with my doubts and questions.

Reply:
Woah! You did great for your first PR! Congratulations🎉 No need to refork. 

I believe, In CSS, this is the part that makes lines toggle work
```css
.paper {
  background-color: unset !important;
  line-height: 1.5em;
  background-size: 100% 1.5em;
  padding: unset;
}
.lines {
  background-image: linear-gradient(#999 0.05em, transparent 0.1em);
  background-color: unset !important;
  line-height: 1.5em;
  background-size: 100% 1.5em;
  padding: unset;
}
```
You can copy-paste CSS from my current master and then add the above two lines and commit to your master. 

Committing to your master will automatically reflect those changes in the PR too.

Let me know if you have time limitations, in that case, I can fix them as well.

It's ok to let HTML and JS be this way. 

and you did ask the fair question. My first PR was mess lol you did so good! 🌻

Reply:
OK Now I have pushed again does it works.